src/hotspot/os/linux/os_linux.cpp

```diff
@@ -170,6 +170,9 @@ const char * os::Linux::_libc_version = NULL;
 const char * os::Linux::_libpthread_version = NULL;
 size_t os::Linux::_default_large_page_size = 0;
 
+void* os::Linux::_mallinfo_fun_ptr = NULL;
+void* os::Linux::_mallinfo2_fun_ptr = NULL;
```

Comment:
Should this be ifdef __GLIBC__ too?

Reply:
Yes, unless you want unused-variable warnings on Alpine.